config/config.mk

```diff
@@ -32,6 +32,9 @@ boot_addr ?= 2684354560 # A0000000
 l2_base  ?= 2147483648 # 80000000
 l2_size  ?= 4194304    # 400000
 l2_banks ?= 4
+ifeq ($(config), terapool)
+  l2_banks = 16
+endif
```

Comment:
This should not be necessary if you have set the l2 banks in `terapool.mk` already. The specific config will override the default from this file.